deepfence_frontend/apps/dashboard/src/features/integrations/components/integration-form/utils.ts

```diff
@@ -0,0 +1,165 @@
+import {
+  ModelIntegrationFilters,
+  ModelNodeIdentifierNodeTypeEnum,
+} from '@/api/generated';
+import { ScanTypeEnum } from '@/types/common';
+
+export const IntegrationType = {
+  slack: 'slack',
+  pagerDuty: 'pagerduty',
+  email: 'email',
+  httpEndpoint: 'http_endpoint',
+  microsoftTeams: 'teams',
+  splunk: 'splunk',
+  sumoLogic: 'sumologic',
+  elasticsearch: 'elasticsearch',
+  googleChronicle: 'googlechronicle',
+  awsSecurityHub: 'aws_security_hub',
+  jira: 'jira',
+  s3: 's3',
+} as const;
+
+export function getIntegratinPrettyName(type: string) {
```

Comment:
instead of string type, we can use `(typeof IntegrationType)[keyof typeof IntegrationType]` and use exhaustive check.